core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -748,13 +747,14 @@ private SqlNode createSqlWindowBound(RexWindowBound rexWindowBound) {
     }
 
     private SqlNode createLeftCall(SqlOperator op, List<SqlNode> nodeList) {
-      if (nodeList.size() == 2) {
-        return op.createCall(new SqlNodeList(nodeList, POS));
+      assert nodeList != null && nodeList.size() >= 2;
```

Comment:
just some minor notes; code is fine.

* the summary of the commit headline should reflect more what the actual issue was and not describe the actual fix 
 - there are some guidelines here: https://calcite.apache.org/develop/ I would peek into `git log --grep Exception` to read a few examples
* I feel that this comment should really be in the body of the commit message
* this assert (if it fires) will just tell the user that something went wrong. You can either add a message or remove it as it seems like it works like this "by design"

Reply:
Just notice your comments, sorry for being late.
Thanks for your review, I will take care for future PRs.